<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableWindow</class>
 <widget class="QMainWindow" name="TableWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a layout</string>
  </property>
  <property name="windowIcon">
   <iconset resource="share/resources/icon.qrc">
    <normaloff>:/icon/64x64/icon64x64.png</normaloff>:/icon/64x64/icon64x64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="VTableGraphicsView" name="view"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionNext"/>
   <addaction name="actionTurn"/>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="actionMirror"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionStop"/>
   <addaction name="actionLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Pages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="iconSize">
        <size>
         <width>200</width>
         <height>200</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Static</enum>
       </property>
       <property name="flow">
        <enum>QListView::TopToBottom</enum>
       </property>
       <property name="isWrapping" stdset="0">
        <bool>true</bool>
       </property>
       <property name="viewMode">
        <enum>QListView::IconMode</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save layout</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="go-next">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="toolTip">
    <string>Next detail</string>
   </property>
  </action>
  <action name="actionTurn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="object-rotate-left">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Turn</string>
   </property>
   <property name="toolTip">
    <string>Turn the detail 90 degrees</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme="process-stop">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop laying</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Enlarge letter</string>
   </property>
   <property name="toolTip">
    <string>Enlarge the length of the sheet</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Reduce sheet</string>
   </property>
   <property name="toolTip">
    <string>Reduce the length of the sheet</string>
   </property>
  </action>
  <action name="actionMirror">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="share/resources/icon.qrc">
     <normaloff>:/icon/16x16/mirror.png</normaloff>:/icon/16x16/mirror.png</iconset>
   </property>
   <property name="text">
    <string>Mirroring</string>
   </property>
   <property name="toolTip">
    <string>Mirroring</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset theme="zoom-in">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset theme="zoom-out">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionLayout">
   <property name="text">
    <string>Layout</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VTableGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>vtablegraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="share/resources/icon.qrc"/>
 </resources>
 <connections/>
</ui>
